### PR TITLE
test_flows for ap-metrics-response as part of the refactoring

### PR DESCRIPTION
**Pull request by RanRegev**
added first few lines:
confing request and checking for ACK
the test fails at the moment

### CODE REVIEW

File: agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp [hunk 1]
```diff
@@ -2201,6 +2208,7 @@ bool backhaul_manager::handle_1905_1_message(ieee1905_1::CmduMessageRx &cmdu_rx,
     }
     case ieee1905_1::eMessageType::MULTI_AP_POLICY_CONFIG_REQUEST_MESSAGE: {
         return handle_multi_ap_policy_config_request(cmdu_rx, src_mac);
+    }
```
Comment (arnout) [inline agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp:2291]: Could you make a small PR with just this fix?

**Comment by RanRegev** [comment]
here:
https://github.com/prplfoundation/prplMesh/pull/1451

**Comment by RanRegev** [comment]
github.com/prplfoundation/prplMesh/pull/1451

**Comment by RanRegev** [comment]
https://github.com/prplfoundation/prplMesh/pull/1451

**Comment by RanRegev** [comment]
This PR is too messy.
Creating a new PR with only two commits.
#1467

**Pull request closed without merging**